src/elements/content-sidebar/activity-feed/comment/__tests__/__snapshots__/Comment.test.js.snap

```diff
@@ -81,135 +81,139 @@ exports[`elements/content-sidebar/ActivityFeed/comment/Comment should correctly
 `;
 
 exports[`elements/content-sidebar/ActivityFeed/comment/Comment should correctly render comment when translation is enabled 1`] = `
-<ForwardRef(ActivityCard)
```

Comment:
These snapshots changes are a little concerning. Do we need to `.dive()` or mock out `react-intl` in the test to get into the component again?

Reply:
I'll need a hand with this specs 
maybe you can point me to a file with `react-intl` mocked or an example of how to deal with `injectIntl`